cdist/conf/type/__sysctl/explorer/conf-path

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# 2018 Darko Poljak (darko.poljak at gmail.com)
+#
+# This file is part of cdist.
+#
+# cdist is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cdist is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with cdist. If not, see <http://www.gnu.org/licenses/>.
+#
+
+[ -d "/etc/sysctl.d" ] && echo "/etc/sysctl.d/99-Z-sysctl-cdist.conf" || echo "/etc/sysctl.conf"
```

Comment:
I just wanted to complain about @asteven 's claim that we always want to be last in this directory, but I can't, he's totally right. I think *if* an existing /etc/sysctl.d means that files are being read in there, then this patch is a beauty.